test/conformance/exp_command_buffer/update/invalid_update.cpp

```diff
@@ -224,3 +224,254 @@ TEST_P(InvalidUpdateTest, InvalidDimensions) {
         UR_RESULT_ERROR_INVALID_VALUE,
         urCommandBufferUpdateKernelLaunchExp(command_handle, &update_desc));
 }
+
+// Tests that an error is thrown when trying to update a kernel capability
+// that isn't supported.
+struct urInvalidUpdateCommandBufferExpExecutionTest
```

Comment:
Nit but is there a reason for using the `ur` prefix here for the fixture name? This differs from all our other tests so would prefer consistency if possible.